packages/common/src/components/UnifiedView/UvDateRangeList/UvDateRangeList.tsx

```diff
@@ -0,0 +1,37 @@
+import React from 'react'
+import { ConfigUeSickLeavePeriod, ValueDateRangeList } from '../../..'
+import { Table, TableBody, TableCell, TableHeader, TableRow } from '../../Table'
+
+export const UvDateRangeList: React.FC<{
+  value: ValueDateRangeList
+  config: ConfigUeSickLeavePeriod
+}> = ({ value, config }) => {
+  if (value.list.length > 0 && value.list.some((val) => val.from && val.to)) {
+    return (
+      <Table>
+        <TableHeader>
+          <TableRow>
+            <TableCell>Nedsättningsgrad</TableCell>
+            <TableCell>Från och med</TableCell>
```

Comment:
Should we use label from BE? 
![image](https://user-images.githubusercontent.com/79509664/214776462-d224612d-1598-41cd-931a-74b5fd236c95.png)

Reply:
DateRangeList is an old valuetype used for the sickLeave component, and in this case we don't get any label from config from backend